testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/TestClassProvider.java

```diff
@@ -0,0 +1,82 @@
+package org.keycloak.testsuite.arquillian;
+
+import com.sun.net.httpserver.HttpContext;
```

Comment:
Will this work on IBM JDK? Maybe embedded undertow is a bit safer option regarding this? See for example class OcspHandler from our testsuite for some details.

Reply:
+1

Reply:
Changing to Undertow